cmake/template/deactivate.bat.in

```diff
@@ -0,0 +1,144 @@
+call :remove_from_env GZ_SIM_RESOURCE_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\gazebo\models"
+call :remove_from_env GZ_SIM_RESOURCE_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\iCub\robots"
+call :remove_from_env GZ_SIM_RESOURCE_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\ergoCub\robots"
+call :remove_from_env GZ_SIM_RESOURCE_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share"
+call :remove_from_env GZ_SIM_RESOURCE_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\gazebo\worlds"
+@endcmakeif ROBOTOLOGY_USES_GZ
+
+@cmakeif ROBOTOLOGY_ENABLE_ROBOT_TESTING
+rem Cleanup ROBOTOLOGY_ENABLE_ROBOT_TESTING-specific lines
+call :remove_from_path PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\lib\robottestingframework"
+call :remove_from_env YARP_DATA_DIRS "@PROJECT_SOURCE_DIR_NATIVE_PATH@\src\icub-tests\suites"
+call :remove_from_env BLOCKTEST_RESOURCE_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\bin"
+@endcmakeif ROBOTOLOGY_ENABLE_ROBOT_TESTING
+
+@cmakeif ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS
+rem Cleanup ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS-specific lines
+call :remove_from_env YARP_DATA_DIRS "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\human-gazebo"
+call :remove_from_env YARP_DATA_DIRS "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\HumanDynamicsEstimation"
+@endcmakeif ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS
+
+@cmakeif ROBOTOLOGY_USES_PYTHON
+rem Cleanup the python bindings directory from the PYTHON_PATH
+call :remove_from_env PYTHONPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\@ROBOTOLOGY_SUPERBUILD_PYTHON_INSTALL_DIR@"
+rem Remove from the PATH where pure python packages install the scripts
+call :remove_from_path PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\@ROBOTOLOGY_SUPERBUILD_PYTHON_INSTALL_DIR@\bin"
+@endcmakeif ROBOTOLOGY_USES_PYTHON
+
+@cmakeif ROBOTOLOGY_USES_MATLAB
+rem Cleanup ROBOTOLOGY_USES_MATLAB-specific lines
+call :remove_from_env YARP_DATA_DIRS "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\RRbot"
+call :remove_from_env GAZEBO_MODEL_PATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\RRbot\robots"
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\mex"
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\mex\+wbc\simulink"
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\mex\+wbc\simulink\MomentumVelocityControl"
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\mex\+wbc\examples"
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\WBToolbox"
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\share\WBToolbox\images"
+@endcmakeif ROBOTOLOGY_USES_MATLAB
+
+@cmakeif ROBOTOLOGY_WITH_MATLAB_AND_MUJOCO
+call :remove_from_env MATLABPATH "%ROBOTOLOGY_SUPERBUILD_INSTALL_PREFIX_LOCAL%\mex\mujoco_simulink_blockset"
+@endcmakeif ROBOTOLOGY_WITH_MATLAB_AND_MUJOCO
+
+goto :eof
```

Comment:
I am a bit confused by this. The function below get defined even if we exit here?